task/sql-tasks.js

```diff
@@ -177,7 +278,13 @@ async function task_1_12(db) {
  *
  */
 async function task_1_13(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT

```

Comment:
In this case it's even better to do this

```
select
(select count()...) as 'T'
(select count()...) as 'TT'
```
notice that from clause is optional

Reply:
> In this case it's even better to do this
> 
> ```
> select
> (select count()...) as 'T'
> (select count()...) as 'TT'
> ```
> 
> notice that from clause is optional

Could you elaborate on what is wrong here? I have the same structure as you wrote:
```
SELECT
	        (SELECT COUNT(ProductName)) AS "TotalOfCurrentProducts",
	        (SELECT SUM(Discontinued)) AS "TotalOfDiscontinuedProducts"
        FROM Products;
```
Maybe I don't understand something